compiler/fir/analysis-tests/testData/resolveWithStdlib/contracts/fromSource/good/returnsImplies/propertyAccessors.kt

```diff
@@ -27,4 +27,4 @@ fun test_1(a: A?) {
 fun test_2(a: A?) {
     a.isNotNull = true
     a.foo()
-}
\ No newline at end of file
+}
```

Comment:
Same

Reply:
This change should not be here. Please revert it (e.g. "Revert selected changes" from Git plugin).